src/Auth0.OidcClient.Android/Auth0Client.cs

```diff
@@ -9,16 +12,98 @@ namespace Auth0.OidcClient
     public class Auth0Client : Auth0ClientBase
     {
         /// <summary>
-        /// Creates a new instance of the Auth0 OIDC Client.
+        /// Create a new instance of <see cref="Auth0Client"/> with a given <see cref="Auth0ClientOptions"/>.
         /// </summary>
-        /// <param name="options">The <see cref="Auth0ClientOptions"/> specifying the configuration for the Auth0 OIDC Client.</param>
+        /// <param name="options">The <see cref="Auth0ClientOptions"/> specifying the configuration to use.</param>
+        /// <remarks>options.RedirectUri must match your <see cref="Activity"/> <see cref="IntentFilterPriority"/>
+        /// DataScheme, DataPathPrefix and DataHost values.
+        /// If not supplied it will presume the convention
+        /// <code>$"{Context.PackageName}://{options.Domain}/android/{Context.PackageName}/callback".ToLower();</code>.
+        /// Your <see cref="IntentFilterAttribute"/> should have DataScheme, DataPathPrefix and DataHost with values that match.
+        /// Alternatively set <see cref="Auth0ClientOptions"/> RedirectUri and PostLogoutRedirectUri to match your <see cref="IntentFilterAttribute"/>. 
+        /// DataScheme must be lower-case or Android will not receive the callbacks.
+        /// </remarks>
         public Auth0Client(Auth0ClientOptions options)
             : base(options, "xamarin-android")
         {
-            options.Browser = options.Browser ?? new AutoSelectBrowser(null);
-            var callbackUrl = $"{Context.PackageName}://{options.Domain}/android/{Context.PackageName}/callback".ToLower();
-            options.RedirectUri = options.RedirectUri ?? callbackUrl;
-            options.PostLogoutRedirectUri = options.PostLogoutRedirectUri ?? callbackUrl;
+            var defaultRedirectUri = options.RedirectUri == null || options.PostLogoutRedirectUri == null
+                ? GetConventionCallbackUri(options.Domain) : null;
+
+            options.RedirectUri = options.RedirectUri ?? defaultRedirectUri;
+            options.PostLogoutRedirectUri = options.PostLogoutRedirectUri ?? defaultRedirectUri;
```

Comment:
Is `PostLogoutRedirectUri` where you would end up after you log out of Auth0? If so, should that be the same as the callback used for logging in (if I understand this correctly)?

Reply:
Yes and in the case of this app that url defaults to just calling the app back which in turn ends up resuming after the await:

e.g.

```csharp
var x = await auth0Client.LogoutAsync();
Console.Log("You are logged out");
```

This app defaults the redirecturls so that flow continues after the await.  But they can alternatively override it with something entirely different.